.env.development

```diff
@@ -0,0 +1,5 @@
+#PORT=8100
+DATADOG_API_KEY=
+NEXT_PUBLIC_DIGS_ENV=
+DATADOG_SERVER_LOGGER_ENABLED=
+INCLUDE_TRACE_IN_LOGS=
```

Comment:
❓ How are these passed in deployed environments? I thought these had to be passed through per-request, like how we currently do `WEAVIATE_HOST`, etc.

Reply:
Yeah can do that or just bake it in at build time in docker compose. Whichever we think is the best route.

Reply:
@pavan4 I don't think we can bake them in since Hasura builds the image for us (the reason we have this all in a public repo). It's an option for v3 data connectors but until we make the switch we need to pass them per-reques.t

Reply:
Yeah we can ask hasura to bake this in - https://hasura.io/docs/latest/hasura-cli/connector-plugin/commands/connector_create/ just pass all the env vars through the CLI. Or did I misread your question?

Reply:
Ah, didn't know that! Will be nice to bake in for some of these, but we'll still run into the problem of all envs hitting the same GDC (since we're pretty limited on # of active deployments).

So we'll need to move `NEXT_PUBLIC_DIGS_ENV` to a pre-request basis, but the rest can stay 👍